Packages/StatusKit/Sources/StatusKit/Row/StatusRowView.swift

```diff
@@ -219,6 +233,23 @@ public struct StatusRowView: View {
       StatusDataControllerProvider.shared.dataController(for: viewModel.finalStatus,
                                                          client: viewModel.client)
     )
+    .alert("alert.translation-error.deepl", isPresented: $viewModel.deeplTranslationError) {
```

Comment:
What I was saying about localization is that for the new stuff you want to put in, you can put the English inline directly, it'll become the key and also the default localization for every languages not providing a translation.